client/gutenberg/extensions/map-block/settings.js

```diff
@@ -0,0 +1,406 @@
+					stylers: [
+						{
+							visibility: 'off',
+						},
+					],
+				},
+				{
+					featureType: 'transit',
+					elementType: 'labels.icons',
+					stylers: [
+						{
+							visibility: 'off',
+						},
+					],
+				},
+				{
+					featureType: 'transit',
+					elementType: 'labels.text',
+					stylers: [
+						{
+							visibility: 'off',
+						},
+					],
+				},
+			],
+		},
+		satellite: {
+			map_type: 'SATELLITE',
+			styles: [
+				{
+					elementType: 'labels',
+					stylers: [
+						{
+							visibility: 'on',
+						},
+					],
+				},
+				{
+					featureType: 'poi',
+					elementType: 'labels.text',
+					stylers: [
+						{
+							visibility: 'simplified',
+						},
+					],
+				},
+				{
+					featureType: 'poi',
+					elementType: 'labels.icon',
+					stylers: [
+						{
+							visibility: 'off',
+						},
+					],
+				},
+				{
+					featureType: 'transit',
+					elementType: 'labels.icons',
+					stylers: [
+						{
+							visibility: 'off',
+						},
+					],
+				},
+				{
+					featureType: 'transit',
+					elementType: 'labels.text',
+					stylers: [
+						{
+							visibility: 'off',
+						},
+					],
+				},
+			],
+		},
+		satellite_with_features: {
+			map_type: 'HYBRID',
+			styles: [
+				{
+					elementType: 'labels',
+					stylers: [
+						{
+							visibility: 'on',
+						},
+					],
+				},
+				{
+					featureType: 'poi',
+					elementType: 'labels.text',
+					stylers: [
+						{
+							visibility: 'simplified',
+						},
+					],
+				},
+				{
+					featureType: 'poi',
+					elementType: 'labels.icon',
+					stylers: [
+						{
+							visibility: 'off',
+						},
+					],
+				},
+				{
+					featureType: 'transit',
+					elementType: 'labels.icons',
+					stylers: [
+						{
+							visibility: 'off',
+						},
+					],
+				},
+				{
+					featureType: 'transit',
+					elementType: 'labels.text',
+					stylers: [
+						{
+							visibility: 'off',
+						},
+					],
+				},
+			],
+		},
+		terrain: {
+			map_type: 'TERRAIN',
+			styles: [
+				{
+					featureType: 'administrative',
+					stylers: [
+						{
+							visibility: 'off',
+						},
+					],
+				},
+				{
+					elementType: 'labels',
+					stylers: [
+						{
+							visibility: 'off',
+						},
+					],
+				},
+				{
+					featureType: 'poi',
+					stylers: [
+						{
+							visibility: 'simplified',
+						},
+					],
+				},
+				{
+					featureType: 'poi',
+					elementType: 'labels',
+					stylers: [
+						{
+							visibility: 'off',
+						},
+					],
+				},
+				{
+					featureType: 'road',
+					stylers: [
+						{
+							visibility: 'off',
+						},
+					],
+				},
+				{
+					featureType: 'transit',
+					stylers: [
+						{
+							visibility: 'off',
+						},
+					],
+				},
+				{
+					featureType: 'water',
+					elementType: 'labels',
+					stylers: [
+						{
+							visibility: 'off',
+						},
+					],
+				},
+			],
+		},
+		terrain_with_features: {
+			map_type: 'TERRAIN',
```

Comment:
Map type is the same as above.

Reply:
My guess is this is intentional. Just wanted to bring it up in case it wasn't. The same goes for the map_type below.

Reply:
Yes, this is intentional. The map type is the same, but the difference is in the labels and a couple of other parameters.